mirrord/layer/src/file/hooks.rs

```diff
@@ -503,6 +504,21 @@ unsafe extern "C" fn fill_stat(out_stat: *mut stat, metadata: &MetadataInternal)
     out.st_blocks = best_effort_cast(metadata.blocks);
 }
 
+/// Fills the `stat` struct with the metadata
+unsafe extern "C" fn fill_statfs(out_stat: *mut statfs, metadata: &FsMetadataInternal) {
+    out_stat.write_bytes(0, 1);
```

Comment:
Add a comment telling why we need this `write_bytes`.

Reply:
TBH I'm not sure if it is needed I just started by copying `fill_stat`

Reply:
It looks like it works without it too